examples/common/python/connectors/direct/tcs_listener/tcs_worker_registry_handler.py

```diff
@@ -135,7 +135,7 @@ def __validate_input_worker_status(self, params):
         try:
             worker_status = params["status"]
             status = int(worker_status)
-            wo_status = WorkerStatus(status)
+            WorkerStatus(status)
```

Comment:
Why do we need this -> WorkerStatus(status)

Reply:
this is to check if status value is within the value supported by enum WorkerStatus.